src/list.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable */
+import 'lodash';
```

Comment:
It's better to remove any imports or codes that are not being used, in accordance with the YAGNI principle.